cogs/welcome/welcome.py

```diff
@@ -28,8 +31,22 @@ class Welcome(commands.Cog):  # pylint: disable=too-many-instance-attributes
     def __init__(self, bot: Red):
         self.bot = bot
         self.config = Config.get_conf(self, identifier=5842647, force_registration=True)
+
         self.config.register_guild(**DEFAULT_GUILD)
 
+        self.data_dir = data_manager.cog_data_path(cog_instance=self)
+        self.img_dir = os.path.join(self.data_dir, "welcome_imgs")
+
+        # create folder to hold welcome images
+        try:
+            os.mkdir(self.img_dir)
```

Comment:
Consider using [`Path.makedir()`](https://docs.python.org/3/library/pathlib.html#pathlib.Path.mkdir).
```suggestion
            self.img_dir.mkdir(parents=True, exist_ok=True)
```